### PR TITLE
Change UI and Color of Page and To-Do Cards

### PR DESCRIPTION
**Pull request by AmreshSinha**
#19 

Screenshot_20211004_185203
Changed some extra things too. Color Palette took from Duck Duck go night theme. If you find any unsatisfaction then you can reply back with the edits or feel you want :D I will try experimenting with color palettes and get a better one.
And if everything is good then you can merge it :rocket: (don't forget for hacktoberfest-accepted label :blush: )

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -122,12 +122,12 @@
 /* Dark theme */
 
 .dark {
-  background-color: #121212 !important;
+  background-color: #1C1C1C !important;
```
Comment (nicosh) [inline src/App.css]: leave it as it is

File: src/App.css [hunk 2]
```diff
@@ -139,16 +139,16 @@
...
 .dark .ant-select:not(.ant-select-customize-input) .ant-select-selector {
-  background-color: #272727 !important;
+  background-color: #333333 !important;
 }
 
 .dark .ant-select-selection-item {
-  background-color: rgb(94, 94, 94) !important;
-  border: #272727 !important;
+  background-color: #1890FF !important;
+  border: #1890FF !important;
```
Comment (nicosh) [inline src/App.css]: leave it as it is

File: src/App.css [hunk 3]
```diff
@@ -139,16 +139,16 @@
 }
 
 .dark .ant-select-selector {
-  background-color: #272727 !important;
+  background-color: #333333 !important;
```
Comment (nicosh) [inline src/App.css]: leave it as it is

**Comment by nicosh** [comment]
@AmreshSinha thank you but the issue was related to something else :

Cattura
The cards inside todo lists still be white.
You can discard all  other changes   except the last one 
```
.dark .react-kanban-column {
  border-radius: 10px;
  background-color: #121212!important;
}

```

**Comment by AmreshSinha** [comment]
ohh. I don't know why but the card was normal my side. Okay I will look again into it

**Comment by nicosh** [comment]
@AmreshSinha thank you! also remember to revert other changes!

**Comment by AmreshSinha** [comment]
> @AmreshSinha thank you! also remember to revert other changes!

yeah I have resolved the above reviews :)

**Comment by AmreshSinha** [comment]
Screenshot_20211004_192601


See there's no white card :/

**Comment by AmreshSinha** [comment]
ohh my bad

**Comment by nicosh** [comment]
@AmreshSinha click on `+` and add a new item :)

**Comment by AmreshSinha** [comment]
Screenshot_20211004_201643
How's this?

**Comment by nicosh** [comment]
@AmreshSinha it's ok, please first resolve the pending reviews so i can merge :)

**Comment by AmreshSinha** [comment]
I have resolved those reviews earlier when you said to leave it as it is. I am making another commit.

**Comment by AmreshSinha** [comment]
bro revert. I had to make another commit :disappointed: the recent change

**Comment by AmreshSinha** [comment]
Should I make another PR?

**Comment by nicosh** [comment]
@AmreshSinha yes please

**Pull request merged**